### PR TITLE
docs: add playground to Storybook

### PR DESCRIPTION
**Pull request by rodleyorosa**
Refs: CDS-302

### CODE REVIEW

**Review by beawar — COMMENTED** [review]
Remove the old playground file

File: src/playground/playground-helper.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      </Container>
+    )
+}`;
+
+const css = `* {
+    margin: 0;
+    padding: 0;
+}`;
+
+export default { jsx, css };
```
Comment (beawar) [inline src/playground/playground-helper.ts]: I would prefer the named export instead of the default one, but it's just my 2 cents

**Review by beawar — DISMISSED** [review]
I feel we can improve the playground with the autocompletion and more, but it is also a good start point, as is. We can continue to improve it while migrating the other components

**Comment by beawar** [comment]
@rodleyorosa  the commit should be marked as docs, since we are doing something for the documentation

**Review by beawar — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Pull request merged**